apps/backend/src/authn/authn.controller.ts

```diff
@@ -104,26 +104,31 @@ export class AuthnController {
   async loginToOkta(
     @Req() req: Request
   ): Promise<{userID: string; accessToken: string}> {
+    console.log('in the okta login func');
+    console.log(JSON.stringify(req.session, null, 2));
     return this.authnService.login(req.user as User);
   }
 
-  @Get('okta/callback')
+  @Get('okta_callback')
```

Comment:
I though the path for the get (`okta_callback`) was related to the path for the AuthGuard (`okta`) but George told me it isn't. Mostly just noting this here for understanding purposes.

Reply:
They are entirely independent urls.  The /okta url's more or less sole purpose is to run into the guard that says you gotta log in via okta to hit it and replaces our implementation for that endpoint with the stuff that kicks off the auth process.  The /okta_callback url is what we supply to okta as a callback back into our application, has one last check to confirm that we're actually logged in w/ the state appropriately set and all, and then we can go off and create our user within the app.